app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt

```diff
@@ -208,6 +209,30 @@ class PoliciesActivityTest {
     }
   }
 
+  @Test
+  fun testPoliciesActivity_withArabicSystemLocale_contentIsInEnglishAndLeftAligned() {
```

Comment:
This test doesn't actually seem to be setting up an Arabic system locale. Did you verify that this fails correctly with your fix not in place (i.e. reverting the change to `PoliciesActivity` such that it uses the system language instead of forcing English)?